pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py

```diff
@@ -209,7 +209,8 @@ def handle_api_question_and_reply(self, data):
                     data['url'])
             return
         house = {
-            'National Assembly': 'N'
+            'National Assembly': 'N',
+            'National Council of Provinces': 'C'
         }.get(data['house']['name'])
         if not house:
             print "Skipping {} because the house {} is not supported.".format(
```

Comment:
it feels like time to factor out the logic in handle_api_question_and_reply - it's a loooooooong method and does a whole bunch of parsing/translation between pmg and pombola data formats